qiime2/sdk/tests/test_artifact.py

```diff
@@ -496,7 +502,15 @@ def test_artifact_validate_max(self):
         self.assertTrue(True)  # Checkpoint assertion
         A = Artifact.import_data('IntSequence1', [1, 2, 3, 4, 5, 6, 7, 10])
         with self.assertRaisesRegex(ValidationError, '3 more'):
-            A.validate('max')
+            A.validate(level='max')
+
+    def test_artifact_validate_max_on_import(self):
+        A = Artifact.import_data('IntSequence1', [1, 2, 3, 4, 5, 8])
+        A.validate('min')
+        self.assertTrue(True)  # Checkpoint assertion
+        fp = get_data_path('intsequence-fail-max-validation.txt')
```

Comment:
I would recommend adding
```python
fmt = IntSequenceFormat(fp)
fmt.validate(level='min')
self.assertTrue(True)  # Checkpoint assertion
```

As this demonstrates that the file is valid at a minimum level, therefore the error raised on import _must_ be because it is validating max now.